### DISCUSSION TITLE
ADI longitudinally?

### DISCUSSION THREAD

**Discussion by mangstad** [now-i-know-my-abcd/docs#56] [Where in the World is…? (Help)] [answered]
Are the linked external variables (like ADI) going to be available longitudinally? Obviously some number of participants move during the study, so variables like these relating to where they live will change, but so far it seems they only have baseline data provided for the linked external data files.

**Comment by kharloews** [comment, marked as answer]
Hi hi! I can definitely answer this one! That is correct, variables based on geocoded residential addresses are only available for baseline. Other time points will probably not be available until about the 7.0 release.

There are a couple of limiting factors: The biggest one is that the SOP for collecting residential information from participants was changed recently, since we found some issues with the initial residential history-collecting SOP, which then impacted the validity of the data following that of baseline. Since it takes almost two years for the entire sample to go through a round of in-person visits, it will be a while before we can get these more certain estimates of both retrospective (where they lived from birth to the start of the study) and prospective residential histories (where they have lived since the study has started). 

The second factor is deciding how to link subsequent versions of a construct and how often that should be, especially when we consider all of the variables now linked to ABCD. For the ADI, there is a tough choice to make: the next available time point for the ADI available from the Neighborhood Atlas actually changes some of the variables that make up the total weighted sum score (# of telephones and plumbing)...should ABCD just plug in the new version, even though the items are now different (even if they reflect a later time window?), or should we manually compute the ADI measure ourselves from the ACS estimates while preserving the same variables? Each choice has its drawbacks (including time, the working group can only accomplish so much at a time!), and you can imagine how this gets exponentially more difficult when you consider every single dataset. 

But all of that to say...we're working on it! It is frustrating, as countless studies could benefit from these data, but we want to make sure we can speak to the validity of the data! :)

Hope that's useful! 
Carlos Cardenas-Iniguez
Co-chair, LED-Environment and Policy Working Group

**Reply by mangstad** [reply]
Thanks for the detailed info. Very enlightening.
